### ISSUE TITLE
Installation fails on Ubuntu / venv

### ISSUE THREAD

**Issue by nrathaus** [OWASP/OFFAT#88]
Recreate:
```
mkdir /tmp/test
cd /tmp/test
python3 -mvenv venv
source venv/bin/activate
pip3 install offat
```

Results in:
```
ERROR: Could not find a version that satisfies the requirement offat (from versions: none)
ERROR: No matching distribution found for offat
```

**Comment by nrathaus** [comment]
Maybe related to this:
```
python = "^3.11"
```

Why is 3.11 required?

**Comment by dmdhrumilmistry** [comment]
As discussed in this PR https://github.com/OWASP/OFFAT/pull/89, I'll test tool for python 3.10 over this weekend and add support if everything seems to be working fine.

I'll be keeping this issue open until I test it.

**Comment by dmdhrumilmistry** [comment]
I have tested tool's compatibility with python 3.10 it seems to be working fine. 3.10 compatibility will be released in `v0.17.4`.
